### PR TITLE
helm3 support for NOTES.txt

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

i added helm 3 commands for NOTES.txt.

### CODE REVIEW

**Comment by developer-guy** [comment]
@ishustava hi, how can i request a reviewer for my PR, can you help me please ?

File: templates/NOTES.txt [hunk 1]
```diff
@@ -7,11 +7,18 @@ Consul with Kubernetes available here:
 https://www.consul.io/docs/platform/k8s/index.html
 
 
-Your release is named {{ .Release.Name }}. To learn more about the release, try:
+Your release is named {{ .Release.Name }}. To learn more about the release,
+
+If you are usign Helm version 2, try:
```
Comment (lkysow) [inline templates/NOTES.txt]:
```suggestion
To learn more about the release if you are using Helm 2, run:
```

File: templates/NOTES.txt [hunk 2]
```diff
@@ -7,11 +7,18 @@ Consul with Kubernetes available here:
...
 
   $ helm status {{ .Release.Name }}
   $ helm get {{ .Release.Name }}
 
+If you are using Helm version 3, try:
```
Comment (lkysow) [inline templates/NOTES.txt]:
```suggestion
To learn more about the release if you are using Helm 3, run:
```

File: templates/NOTES.txt [hunk 3]
```diff
@@ -7,11 +7,18 @@ Consul with Kubernetes available here:
 https://www.consul.io/docs/platform/k8s/index.html
 
 
-Your release is named {{ .Release.Name }}. To learn more about the release, try:
+Your release is named {{ .Release.Name }}. To learn more about the release,
```
Comment (lkysow) [inline templates/NOTES.txt]:
```suggestion
Your release is named {{ .Release.Name }}.
```

Comment (developer-guy) [reply templates/NOTES.txt]: Done in commit : 1195a6c413892e82a8968100272845e3b3aa2db9

File: templates/NOTES.txt (hunk 2 above)
Comment (developer-guy) [reply templates/NOTES.txt]: Done in commit : 1195a6c413892e82a8968100272845e3b3aa2db9

File: templates/NOTES.txt (hunk 1 above)
Comment (developer-guy) [reply templates/NOTES.txt]: Done in commit : 1195a6c413892e82a8968100272845e3b3aa2db9

**Comment by lkysow** [comment]
Thanks!

**Pull request merged**